xknx/devices/cover.py

```diff
@@ -161,11 +159,11 @@ def __str__(self):
 
     async def set_short_down(self):
         """Move cover short down."""
-        await self.step.increase()
+        await self.step.decrease()
 
     async def set_short_up(self):
         """Move cover short up."""
-        await self.step.decrease()
+        await self.step.increase()
 
     async def stop(self):
```

Comment:
I'm not sure if `await self.step.increase()` shall be replaces with `await self.step.decrease()` for stopping.